cmake/Modules/ThirdpartyToolchain.cmake

```diff
@@ -81,3 +81,7 @@ if(RAY_BUILD_TESTS OR RAY_BUILD_BENCHMARKS)
 
   add_dependencies(gflags gflags_ep)
 endif()
+
+set(Boost_USE_STATIC_LIBS ON)
+find_package(Boost COMPONENTS system filesystem REQUIRED)
```

Comment:
We'll probably need to build these ourselves in order to compile with fPIC and link statically (might as well do this on Mac and Ubuntu).

Reply:
Right now it is custom compiled on linux and brew installed on mac. We can custom compile on both (in a separate PR).